app/controllers/kudos_controller.rb

```diff
@@ -4,37 +4,29 @@ class KudosController < ApplicationController
   before_action :set_kudo, only: %i[show edit update destroy]
   before_action :authenticate_employee!, except: %i[index show]
   before_action :correct_employee, only: %i[edit update destroy]
-  # GET /kudos
+
   def index
     @kudos = Kudo.all
   end
 
-  # GET /kudos/1
   def show; end
 
-  # GET /kudos/new
   def new
-    # @kudo = Kudo.new
-    @kudo = current_employee.kudos.build
+    @kudo = current_employee.given_kudos.build
   end
 
-  # GET /kudos/1/edit
   def edit; end
 
-  # POST /kudos
-
   def create
     @kudo = Kudo.new(kudo_params)
```

Comment:
this line is unnecessary - you are overwriting `@kudo` in the next line anyway

Reply:
Fixed.